bin/oneoff/wipe_poste_deliveries_and_contacts_emails

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+# This script removes plaintext emails from the contacts and psote_deliveries
+# tables for all and young Code Studio students, respectively.
+
+require_relative '../../deployment'
+require_relative '../../lib/cdo/db'
+
+# Populate a hash of hashed_emails to birthdays.
+puts 'Populating hashed_email_to_birthday...'
+hashed_email_to_birthday = {}
+DASHBOARD_DB[:users].
+  where(user_type: 'student').
+  where('hashed_email IS NOT NULL').
+  paged_each(rows_per_fetch: 10_000) do |user|
+  hashed_email_to_birthday[user[:hashed_email]] = user[:birthday].strftime('%F')
+end
+
+# Wipe the plaintext email of young students.
```

Comment:
Why only wipe young student emails from contacts, where we wipe all student emails elsewhere?